.github/workflows/run-e2e-tests.yml

```diff
@@ -53,11 +53,11 @@ jobs:
       - name: Setup NodeJS
         uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
         with:
-          node-version: ${{ matrix.version }}
+          node-version: '20'
```

Comment:
Should this be `${{ matrix.version }}`?

Reply:
See [here](https://github.com/aws-powertools/powertools-lambda-typescript/pull/1790#discussion_r1392204289) for more details.